### ISSUE TITLE
🔗 🐞 Bug Report: Readabilty Issues in Dark Mode with Editor Dropdowns (Light Color Background) (Merged with #22430)

### ISSUE THREAD

**Issue by DonSYS91** [elementor/elementor#22514]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I started observing readability issues on latest version of Elementor and Elementor Pro when the OS is set on dark mode, the readability issue is specifically on the dropdowns in the editor (light color dropdown background on light colored font)

There was an issue in the past opened for that case: #21591 now it got better since that version but the issue is still on the dropdown on Windows browsers:
![image](https://github.com/elementor/elementor/assets/18079656/3056387e-517b-4cac-acf1-1ffd3253e26b)

I tested on my Mac with both light and dark system themes and can't reproduce, I can only reproduce on Windows browsers (Chromium-based and Firefox)

When switching the system back to light theme the readability of dropdowns enhances.

What I already tried:
Clean websites only "Hello theme" activated
Disabled all plugins except Elementor
Cleared browser cache / incognito

Versions:
```
WordPress: 6.2.2
PHP: 8.2.5
Elementor Plugin: 3.13.2
Elementor Pro Plugin: 3.13.1
```

### Steps to reproduce

1. Open any WordPress page that was built with Elementor.
2. Hit "edit with Elementor" button in the admin pane
3. Navigate to any element that has a dropdown and click on the dropdown.
4. light background on light colored font is then to be seen as in the screenshot in the description.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: mariadb.org binary distribution v10.11.2-MariaDB-1:10.11.2+maria~deb11
	PHP Version: 8.2.5
	PHP Memory Limit: 3500M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 750M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: REDACTED
	Home URL: https://REDACTED
	WP Multisite: No
	Max Upload Size: 750 MB
	Memory limit: 40M
	Max Memory limit: 3500M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 1
	Admin Email: admin@roblens.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.7.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	CMP - Coming Soon & Maintenance Plugin
		Version: 4.1.8
		Author: NiteoThemes

	CookieYes | GDPR Cookie Consent
		Version: 3.0.9
		Author: CookieYes

	Copy & Delete Posts
		Version: 1.3.9
		Author: Copy Delete Posts

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Header & Footer Builder
		Version: 1.6.14
		Author: Brainstorm Force, Nikhil Chavan

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	Envato Elements
		Version: 2.0.12
		Author: Envato

	Essential Addons for Elementor
		Version: 5.7.2
		Author: WPDeveloper

	Essential Addons for Elementor - Pro
		Version: 5.4.9
		Author: WPDeveloper

	EventON
		Version: 4.3.5
		Author: AshanJay

	LiteSpeed Cache
		Version: 5.4
		Author: LiteSpeed Technologies

	Newsletter, SMTP, Email marketing and Subscribe forms by Sendinblue
		Version: 3.1.61
		Author: Sendinblue

	Rank Math SEO
		Version: 1.0.115
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.36
		Author: Rank Math

	Stratum - Elementor Widgets
		Version: 1.3.14
		Author: MotoPress

	Templately
		Version: 2.2.0
		Author: Templately

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	
	google_maps: Active
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 4 of 4PHP: 2023-04-30 18:23:39 [notice X 1][/usr/local/lsws/REDACTED/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-30 18:24:07 [notice X 13][/usr/local/lsws/REDACTED/wp-content/plugins/elementor-pro/modules/screenshots/module.php::32] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-02 11:09:53 [warning X 3][/usr/local/lsws/REDACTED/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 20:36:12 [warning X 2][/usr/local/lsws/REDACTED/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 13 of 13JS: 2023-04-30 18:31:25 [error X 7][https://REDACTED/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.12.2:2:5009] Cannot read properties of undefined (reading 'attributes') 
JS: 2023-05-03 05:28:15 [error X 9][https://REDACTED/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:632658] Cannot read properties of undefined (reading 'isDesignable') 
JS: 2023-05-03 09:39:33 [error X 16][https://REDACTED/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:731164] _.findWhere is not a function 
JS: 2023-05-04 19:42:35 [error X 28][https://REDACTED/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:785741] Cannot read properties of undefined (reading 'localized') 
JS: 2023-05-04 20:16:27 [error X 1][https://REDACTED/wp-content/plugins/stratum/assets/js/frontend.min.js?ver=1.3.14:1:38563] Cannot read properties of undefined (reading 'center') 
JS: 2023-05-06 18:57:50 [error X 8][https://REDACTED/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:1081263] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-05-06 20:55:47 [error X 2][https://REDACTED/wp-content/uploads/essential-addons-elementor/eael.js?ver=1683405485:16:420196] Cannot read properties of undefined (reading 'top') 
JS: 2023-05-07 21:26:07 [error X 2][https://REDACTED/wp-content/uploads/essential-addons-elementor/eael.js?ver=1683311339:16:377484] Failed to execute 'observe' on 'ResizeObserver': parameter 1 is not of type 'Element'. 
JS: 2023-05-07 21:33:59 [error X 8][https://REDACTED/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.12.3:2:63893] Cannot convert undefined or null to object 
JS: 2023-05-08 05:44:56 [error X 6][https://REDACTED/wp-content/uploads/essential-addons-elementor/eael.js?ver=1683484277:11:439558] a.replace is not a function 
JS: 2023-05-08 19:27:39 [error X 5][https://REDACTED/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.0:3:794970] Cannot read properties of undefined (reading 'toLowerCase') 
JS: 2023-05-10 09:10:15 [error X 1][https://REDACTED/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.0:3:662755] elementorFrontend.elements.window.jQuery is not a function 
JS: 2023-05-10 09:10:15 [error X 1][https://REDACTED/wp-content/uploads/essential-addons-elementor/eael.js?ver=1683713364:16:471980] Cannot read properties of undefined (reading 'hasOwnProperty') 

Log: showing 20 of 252023-05-08 19:45:58 [info] elementor::elementor_updater Started 
2023-05-08 19:45:58 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-08 19:45:58 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-08 19:45:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.0',
)]
2023-05-08 19:45:58 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.0',
)]
2023-05-10 23:19:45 [info] elementor::elementor_updater Started 
2023-05-10 23:19:45 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-10 23:19:45 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-10 23:19:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-10 23:19:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-11 23:22:16 [info] elementor-pro::elementor_pro_updater Started 
2023-05-11 23:22:16 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-11 23:22:16 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-11 23:22:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-11 23:22:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-11 23:22:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-11 23:22:20 [info] elementor::elementor_updater Started 
2023-05-11 23:22:20 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-11 23:22:20 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-11 23:22:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Header & Footer Builder: Compatible
	Elementor Pro: Compatible
	Envato Elements: Incompatible
	Essential Addons for Elementor: Compatible
	Essential Addons for Elementor - Pro: Compatible
	Stratum - Elementor Widgets: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	
	Elementor Header & Footer Builder: Compatible
	Envato Elements: Incompatible
	Essential Addons for Elementor: Compatible
	Essential Addons for Elementor - Pro: Compatible
				

````

</details>

**Comment by babakfp** [comment]
Use this for now:
https://github.com/orgs/elementor/discussions/21826#discussioncomment-5957802
If you take a look at the source code, you can find the solution for that select box issue. I couldn't get the Elementor source code working on my pc so for that reason I couldn't open a PR...
